apps/directory/src/main.js

```diff
@@ -20,6 +20,9 @@ if (import.meta.env.DEV) {
 import "@fortawesome/fontawesome-free/css/all.css";
 import "@fortawesome/fontawesome-free/js/all.js";
 
+/** Add bootstrap icons */
```

Comment:
unnecessary comment, both that it's bootstrap and icons is in the filename

Reply:
The font awesome icons also have a comment above. I simply followed the flow of the project.